package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "ipfs-webui",
-  "version": "0.2.1",
+  "version": "0.3.0",
   "description": "A frontend for IPFS",
-  "main": "js/main.jsx",
   "dependencies": {
     "bootstrap": "3.3.4",
     "d3": "3.5.5",
     "debug": "2.1.3",
     "font-awesome": "4.3.0",
-    "ipfs-api": "1.1.4",
+    "ipfs-api": "2.1.5",
```

Comment:
might want to bump to 2.1.6 if https://github.com/ipfs/node-ipfs-api/pull/45 merges first